sdk/core/azure-core/CMakeLists.txt

```diff
@@ -32,3 +32,5 @@ add_library (Azure::Core ALIAS ${TARGET_NAME})
 
 target_include_directories(${TARGET_NAME} PUBLIC ${CURL_INCLUDE_DIRS})
 target_link_libraries(${TARGET_NAME} PRIVATE CURL::libcurl)
+
+generate_documentation(${TARGET_NAME} 1.0.0-preview.1)
```

Comment:
The version should come from version.h

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/217